login.js

```diff
@@ -0,0 +1,32 @@
+const passwordElem = document.getElementById("selected-user-password");
+const loginElem = document.querySelector(".login-btn");
+const selectElem = document.getElementById("select");
+
+let registeredUsers = JSON.parse(localStorage.getItem("MB_USER_ACCOUNTS"));
+for (let i = 0; i < registeredUsers.length; i++) {
+  let optionElem = document.createElement("option");
+  let text = document.createTextNode(registeredUsers[i].accountName);
```

Comment:
Ditto on keyword

Reply:
You don't need to create text node here. Change to:
`const text = registeredUsers[i].accountName;`